dev/form-layout-auto-responsive.html

```diff
@@ -49,7 +49,7 @@
         render() {
           return html`
             <h1>autoResponsive</h1>
-            <vaadin-form-layout auto-responsive auto-rows max-columns="2">
```

Comment:
Let's add a separate example for minColumns. Applying max-columns 3 to existing examples doesn't display well on wide screens.